src/bond-sdk-cards.js

```diff
@@ -513,6 +513,7 @@ class BondCards {
       data: {
         card_id: cardId,
         new_pin: newPin,
+        current_pin: currentPin,
```

Comment:
I think we should `omit` the `current_pin` field if the `currentPin` is `undefined or null`

Reply:
if its undefined it won't be send in the request